tests/unit/orchestrator/test_models.py

```
@@ -0,0 +1,83 @@
+        assert result.success is False
+        assert result.error == "Connection timeout"
+        assert result.summary_metrics == {}  # Defaults to empty dict
+        assert result.artifacts_path == Path("/tmp/run_0002")
+
+    def test_run_result_with_none_metrics(self):
+        """Test RunResult with None summary_metrics."""
+        result = RunResult(
+            label="run_0003",
+            success=True,
+            summary_metrics=None,
+            artifacts_path=Path("/tmp/run_0003")
```

Ruff (0.14.14)

[error] Probable insecure usage of temporary file or directory: "/tmp/run_0003"

(S108)